src/components/Tokens/TokenDetails/BalanceSummary.tsx

```diff
@@ -53,6 +82,9 @@ export default function BalanceSummary({ tokenAmount, nativeCurrencyAmount, isNa
   const balanceUsdValue = useStablecoinValue(tokenAmount)?.toFixed(2)
   const nativeBalanceUsdValue = useStablecoinValue(nativeCurrencyAmount)?.toFixed(2)
 
+  const { chainName } = useParams<{ chainName?: string }>()
+  const pageChainName = validateUrlChainParam(chainName).toLowerCase()
```

Comment:
nit: if this component is being rendered, I think we can trust the chainName from the params is valid, so we can skip validating and re-lowercasing